.eslintrc.js

```diff
@@ -6,6 +6,8 @@ module.exports = {
   parser: "@typescript-eslint/parser",
   ignorePatterns: [
     "services/minespace-web/webpack.config.ts",
+    "services/minespace-web/cypress.config.ts",
+    "services/minespace-web/cypress/**",
```

Comment:
I am curious how many or what kind of eslint errors came up in the cypress/** directory! Would be nice to have it yell at us when we forget to remove an unused import or use a map function when we want a forEach.